### ISSUE TITLE
Physics Component "over sensitive"

### ISSUE THREAD

**Issue by BlueMagnificent** [urho3d/Urho3D#1054]
I'm using over sensitive here because there is no better term to describe it... I made a table and added different collision shapes to its legs and the top. When I shoot a sphere which has a rigid body through the legs of the table, the table makes a slight movement as if it was brushed by the sphere even though the sphere passed through without contact...
Here is an image showing the collision shapes of the table as designed using thhe Editor:
![table_outlined](https://cloud.githubusercontent.com/assets/6633428/11326336/764ddacc-9167-11e5-8d97-800913b5c911.jpg)

And here is when I shoot a sphere through the legs:
![with_sphere](https://cloud.githubusercontent.com/assets/6633428/11326349/9968fe56-9167-11e5-9ac7-ced65fe546f8.jpg)

I equally noticed something similar in the 13_Ragdolls sample. When a a jack has been shot down, shooting spheres very close to the body but without contact with the collision shapes also gives these effect. I tried this out using  minGW and Visual Studio 2013 (all on windows) and I still got the same effect

**Comment by 1vanK** [comment]
Try to adjust CCD Radius and CCD Motion Threshold

https://www.panda3d.org/manual/index.php/Bullet_Continuous_Collision_Detection

**Comment by JSandusky** [comment]
Are the CCD settings in the RigidBody reasonably set?

**Comment by BlueMagnificent** [comment]
I didn't touch the CCD settings so I believe they still bear the default values, that is, CCD Radius = 0 and CCD Motion Threshold = 0

**Comment by JSandusky** [comment]
CCD settings have to be non-zero for bullet to use it. If you're ball is moving sufficiently fast that'd fit what you're describing, in collision being detected once partly/mostly through the other shape if at all.

**Comment by BlueMagnificent** [comment]
I don't really think its a CCD problem because, as earlier stated, I also noticed this in 13_Ragdolls sample. While testing out this, I added a node collision handler to the table node so as to detect collision. Interestingly I noticed that a node collision was actually being triggered when the sphere passes neatly beneath the table, but for kinematic or infinite mass rigid body setting of the table no collision was triggered. I've included a zip [file](https://www.dropbox.com/s/4rka0p8c3qqkk4s/PhysicsPost.zip?dl=0) containing the test script and assets used just in case there is need to observe it closely.
